packages/editor-ui/src/components/ParameterInputList.vue

```diff
@@ -353,9 +353,6 @@ export default defineComponent({
 				rawValues = get(this.nodeValues, this.path);
 			}
 
-			if (!rawValues) {
-				return false;
-			}
 			// Resolve expressions
```

Comment:
this is fix for error when deleting items from fixed collection when pressing trash icon